app/presenters/api/v2/green_lanes/category_assessment_presenter.rb

```diff
@@ -77,7 +76,8 @@ def certificates
         end
 
         def additional_codes
-          Array.wrap(additional_code)
+          additional_codes = measures.map(&:additional_code).compact
```

Comment:
I'm guessing the answer is yes, but has this association for measures been eager loaded upstream of this call?

Reply:
Yes, here you can see additional_code is in the eager loading plan
https://github.com/trade-tariff/trade-tariff-backend/blob/GL-705_remove_add_code_from_permutation_key/app/services/green_lanes/fetch_goods_nomenclature_service.rb